app/forms/MovementForms.scala

```diff
@@ -112,13 +112,13 @@ object Movement {
   def createMovementRequest(cacheMap: CacheMap,
                             eori: String,
                             choice: Choice): InventoryLinkingMovementRequest = {
-    val ducrForm = cacheMap.getEntry[Ducr](Ducr.id).get
+    val ducrForm = cacheMap.getEntry[Ducr](Ducr.id).getOrElse(Ducr(""))
     val goodsDate =
       cacheMap.getEntry[GoodsDateForm](MovementFormsAndIds.goodsDateId)
     val location =
-      cacheMap.getEntry[LocationForm](MovementFormsAndIds.locationId).get
+      cacheMap.getEntry[LocationForm](MovementFormsAndIds.locationId).getOrElse(LocationForm(None, None, None, None))
```

Comment:
didn't understand the requirement of creating a form if None of the fields are populated.